File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < = 0 || height > 8);


    char block = '#';
    int i = 0;
    while (i < height)
    {
        for (int k = 0; k < height - (i + 1); k++)
        {
            printf(" ");
        }
        for (int j = 0 ; j <= i ; j++)
        {

            printf("%c", block);
        }
        printf("\n");
        i++;
    }

}

File: sentimental-mario-more/mario.py
# TODO

def main():
    height = 0
    while (height <= 0 or height > 8):
        height = input("Height: ")

        try:
            height = int(height)
        except:
            height = 0
            print("Please, write a correct number betweeen 0 and 8")

    i = 1
    while (i <= height ):
        block = "#" * i
        space = " " * (height - i)
        print(f"{space}{block}  {block}")
        i+=1
if __name__ == "__main__":
    main()